histomicstk/preprocessing/color_conversion/lab_mean_std.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+"""Placeholder."""
```

Comment:
Are these placeholders present because they should get updated or because linting fails without them?  If it is a linting issue, we should change the linting rules.

Reply:
These are for linting, but I updated it with a proper description anyways.